### DISCUSSION TITLE
Experimental Offline Standalone Mac Installer for Stable Diffusion Web UI (unofficial) 20230821

### DISCUSSION THREAD

**Discussion by brkirch** [brkirch/stable-diffusion-webui#34] [Announcements]
This is an offline standalone installer for a modified version of AUTOMATIC1111's Stable Diffusion web UI that includes experimental changes and uses the nightly PyTorch build from 8/21. Note that this is an test build so crashes and bugs should be expected. All dependencies needed to start using Stable Diffusion (other than the model) are included and this does not require Xcode command line tools to be installed.

This release currently only has the installer for Apple silicon Macs, an installer for Intel Macs will be included with 1.6.0 RC. macOS 13.4 or later is highly recommended, older versions of macOS are untested.

<hr /><em>This discussion was created from the release <a href='https://github.com/brkirch/stable-diffusion-webui/releases/tag/20230821_experimental'>Experimental Offline Standalone Mac Installer for Stable Diffusion Web UI (unofficial) 20230821</a>.</em>

**Comment by onedoge** [comment]
Hey mate. How do i update? Sorry I'm a noob lol. Tried "git pull" but that didn't work due to how it was installed i think

**Reply by brkirch** [reply]
You'll need to download the new installer and select `Update Existing Install` when you run it.
